101-aks/README.md

```diff
@@ -0,0 +1,3 @@
+# Managed Kubernetes (AKS)
```

Comment:
add a proper readme - check the sample or another in the repo

Reply:
I followed the existing example in 101-acs.

Reply:
Use this one: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/sample-README.md